packages/client/src/components/Navbar/Navbar.component.js

```diff
@@ -4,100 +4,80 @@ import ShoppingCartIcon from '@mui/icons-material/ShoppingCart';
 import FavoriteBorderIcon from '@mui/icons-material/FavoriteBorder';
 import PersonIcon from '@mui/icons-material/Person';
 
+const buttonStyles = {
+  color: '#FFFFFF',
+  fontSize: '1.1rem',
+  fontWeight: '400',
+};
+
 export const Navbar = () => {
   return (
     <AppBar component="nav" width="100%" sx={{ bgcolor: 'mainGreen' }}>
-      <Toolbar
-        fontFamily="Inter"
-        sx={{
-          height: '3rem',
-          justifyContent: 'space-around',
-        }}
-      >
-        <MenuItem>
-          <PersonIcon
-            sx={{ width: '1.75rem', height: '1.75rem', color: 'black' }}
-          />
-        </MenuItem>
-        <Grid item xs={12} align="center">
-          <Box>
-            <img
-              src="https://i.ibb.co/7JGHhKm/image.png"
-              alt="nav bar logo"
-              border="0"
-              width="286"
-              height="98"
+      <Box sx={{ padding: '0.6rem 8.75rem' }}>
```

Comment:
Good job on using rem

Reply:
@babak-f  Thank you so much for your feedback! :)